js/src/contract/item.ts

```diff
@@ -926,13 +930,13 @@ export class ItemProgram {
         amount: args.amount,
       });
 
-    const instructions = [],
+    const instructions : TransactionInstruction[] = [],
       signers = [];
     const itemTransferAuthority =
       accounts.itemTransferAuthority || web3.Keypair.generate();
 
     if (
-      accounts.itemAccount.equals(
+      accounts.itemAccount.equals( // QUESTION: accounts.itemAccount is possibly null - what should we do if it is?
```

Comment:
Just add a check for `null` here.

`accounts.itemAccount && accounts.itemAccount.equals(..`

That should suffice. itemAccount isn't required for this instruction to work.